frontend/controllers/SeasonManageController.coffee

```diff
@@ -86,12 +103,14 @@ angular.module("MagicStick.controllers").controller "SeasonManageController", [
         .error (data) ->
           toastr.error "Couldn't delete match: #{data}"
     $scope.addNextMatchMember = (nextMatchMember, groupId, matchId) ->
-      $http.put("#{matchPath(groupId, matchId)}/members/#{nextMatchMember.id}")
+      member = angular.copy nextMatchMember
```

Comment:
What does cloning the original object get you here? (Not a critisim, I'm ignorant in this regard.)